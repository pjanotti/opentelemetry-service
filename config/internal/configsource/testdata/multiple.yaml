one:
  $cfgsrc/0: deep_0
two:
  $cfgsrc/1:
    key: recurse_to_cfgsrc/0
