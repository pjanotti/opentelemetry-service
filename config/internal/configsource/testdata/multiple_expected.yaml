one:
  deep_0
two:
  deep_1
